### PR TITLE
Add luks disk with slice attribute support

### PR DESCRIPTION
**Pull request by chunfuwen**
File based disk supports slice attribute in luks encryption

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.luks.positive_test.hotplug.device_disk.file_test.slice_enable.encryption_in_source

File: libvirt/tests/cfg/virtual_disks/virtual_disks_luks.cfg [hunk 1]
```diff
@@ -107,6 +107,15 @@
...
                     target_format = "qcow2"
+        - file_test:
+            backend_storage_type = "file"
+            variants:
+                - slice_enable:
+                  preallocation = "full"
+                  slice_support_enable = "yes"
+                  file_image_name = "slice_test.img"
+                  luks_size = "1M"
+                  only positive_test.hotplug.device_disk..encryption_in_source
```
Comment (kylazhang) [inline libvirt/tests/cfg/virtual_disks/virtual_disks_luks.cfg:118]: please use ".." instead of "."

**Review by kylazhang — COMMENTED** [review]
Others are ok for me

File: libvirt/tests/cfg/virtual_disks/virtual_disks_luks.cfg (hunk 1 above)
Comment (chunfuwen) [reply libvirt/tests/cfg/virtual_disks/virtual_disks_luks.cfg]: updated

**Review by kylazhang — APPROVED** [review]

**Pull request merged**